include/psa/crypto.h

```diff
@@ -3781,6 +3781,85 @@ psa_status_t psa_key_derivation_output_key(
  *                          the policy must be the same as in the current
  *                          operation.
  * \param[in,out] operation The key derivation operation object to read from.
+ * \param[in] custom        Customization parameters for the key generation.
+ *                          When this is #PSA_CUSTOM_KEY_PARAMETERS_INIT
+ *                          with \p custom_data_length = 0,
+ *                          this function is equivalent to
+ *                          psa_key_derivation_output_key().
+ * \param[in] custom_data   Variable-length data associated with \c custom.
+ * \param custom_data_length
```

Comment:
This is also not on the same line. I think this style got carried over from some older PR because there's also the same thing on lines 3872 and 4291

Reply:
The content of the left column overflows so the right column starts on the next line, so either we go to the next line or we break alignment. Neither has a clear advantage and we don't follow one style consistently so I'll just leave it alone.